zsh-wakatime.plugin.zsh

```diff
@@ -19,4 +23,8 @@ waka_filename() {
     fi
 }
 
-precmd_functions+=(send_wakatime_heartbeat)
+heartbeat() {
+  (send_wakatime_heartbeat > /dev/null 2>&1 &)
```

Comment:
why we still need a `> /dev/null 2>&1 &`, the `send_wakatime_heartbeat` already has it?